pkgs/development/python-modules/fsspec/default.nix

```diff
@@ -1,28 +1,23 @@
-{ lib
-, buildPythonPackage
-, fetchFromGitHub
-, pythonOlder
-, pytestCheckHook
-, numpy
-, stdenv
-}:
+{ lib, buildPythonPackage, fetchFromGitHub, pythonOlder, pytestCheckHook, numpy
+, stdenv, aiohttp, pytest-vcr, requests }:
```

Comment:
Please revert this change to keep the diff smaller.

Reply:
done.